engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -1,70 +1,70 @@
 <span page-title ng-model="host">{{ 'Content Host' | translate }} {{ host.name }}</span>
 
 <section>
-  <div class="details fl" ng-show="contentHost.id === undefined">
+  <div class="details fl" ng-hide="host.hasContent()">
     <div data-extend-template="common/registration.html"></div>
   </div>
 
-  <div class="details fl" ng-if="contentHost.id">
+  <div class="details fl" ng-show="host.hasContent()">
     <section>
       <h4 translate>Basic Information</h4>
 
       <div class="detail">
         <span class="info-label" translate>Name</span>
         <span class="info-value"
-              bst-edit-text="contentHost.name"
-              readonly="denied('edit_content_hosts', contentHost)"
-              on-save="save(contentHost)">
+              bst-edit-text="host.name"
+              readonly="denied('edit_hosts', host)"
+              on-save="save(host)">
         </span>
       </div>
 
       <div class="detail">
         <span class="info-label" translate>UUID</span>
-        <span class="info-value">{{ contentHost.uuid }}</span>
+        <span class="info-value">{{ host.content.uuid }}</span>
```

Comment:
it doesn't matter too much, but i would use  host.subscription.uuid.  That way we handle the hypervisor case (which does not have a content aspect)